addons/rook/template/base/install.sh

```diff
@@ -160,12 +160,6 @@ function rook_operator_deploy() {
     # upgrade first before applying auth_allow_insecure_global_id_reclaim policy
     rook_maybe_auth_allow_insecure_global_id_reclaim
 
-    # disable bluefs_buffered_io for rook 1.8.x and 1.9.x
-    # See:
-    #   - https://github.com/rook/rook/issues/10160#issuecomment-1168303067
-    #   - https://tracker.ceph.com/issues/54019
-    rook_maybe_bluefs_buffered_io
```

Comment:
What happens if we have installed an old version and we upgrade to this pinned ones with the latest kURL release? Also, from the oldest to the latest should we need to still disabling it?

Reply:
this is in the template used to generate new versions. it is [still](https://github.com/replicatedhq/kURL/blob/6737ccb509692ba58d74fe1dc45482aee579242c/addons/rook/1.9.12/install.sh#L172) in the rook add-on versions affected by the issue. if a user were to upgrade to a later version they would no longer be on a version of ceph affected by the issue (17 quincy).